src/main/java/heronarts/lx/parameter/LXListenableParameter.java

```diff
@@ -39,7 +39,7 @@
 
   private final List<LXParameterListener> listeners = new ArrayList<LXParameterListener>();
 
-  private AggregateParameter parentParameter = null;
+  public AggregateParameter parentParameter = null;
```

Comment:
Use getParentParameter() which was defined on LXParameter.java... setting it to public opens a hole in the API where someone could improperly modify its value.

Reply:
Done